global.json

```diff
@@ -3,7 +3,7 @@
     "dotnet": "7.0.100-preview.7.22377.5",
     "runtimes": {
       "dotnet": [
-        "$(VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion)"
+        "7.0.0-preview.7.22375.6"
```

Comment:
@dsplaisted making this explicitly the preview7 version of runtime for now, since we can't use an 8.0 version yet (I assume)

Reply:
FWIW I've bumped dotnet/winforms and dotnet/wpf to net8 
* https://github.com/dotnet/winforms/pull/7664
* https://github.com/dotnet/wpf/pull/6956

Reply:
Since we can use this now, I will apply the suggestion. Thank you @RussKie